resources/deploy_intermediate_debian.sh

```diff
@@ -0,0 +1,108 @@
+
+mkdir -p "/usr/share/vpn-provisioning/certs"
+cp ./MicrosoftIntuneRootCertificate.cer "/usr/share/vpn-provisioning/certs/MicrosoftIntuneRootCertificate.cer"
+
+mkdir -p "/usr/share/vpn-provisioning/web"
+cp ./index.php "/usr/share/vpn-provisioning/web"
+
+# update hostname
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update vpn name
+sed -i "s/{vpnUrl}/${VPN_URL//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update tenant id
+sed -i "s/{tenantId}/${TENANT_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update application id
+sed -i "s/{applicationId}/${APPLICATION_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update secret application token
+sed -i "s/{secretToken}/${${SECRET_TOKEN//\\/\\\\}//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update admin api token
+sed -i "s/{adminApiToken}/${ADMIN_API_TOKEN}/" "/usr/share/vpn-provisioning/web/index.php"
+
+###############################################################################
+# CERTBOT
+###############################################################################
+
+a2ensite "${INTERMEDIATE_FQDN}"
+systemctl restart apache2
+
+certbot certonly -d "${INTERMEDIATE_FQDN}" --webroot --webroot-path /var/www/html
+
+sed -i "s/#SSLEngine/SSLEngine/" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+sed -i "s/#Redirect/Redirect/" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+
+sed -i "s|#SSLCertificateFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/cert.pem|SSLCertificateFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/cert.pem|" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+sed -i "s|#SSLCertificateKeyFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/privkey.pem|SSLCertificateKeyFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/privkey.pem|" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+sed -i "s|#SSLCertificateChainFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/chain.pem|SSLCertificateChainFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/chain.pem|" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+
+systemctl restart apache2
+
+###############################################################################
+# CRON
+###############################################################################
+
+install -m 600 etc/vpn-provisioning "/etc/vpn-provisioning"
+
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/etc/vpn-provisioning"
+sed -i "s/{vpnUrl}/${VPN_URL//\//\\/}/" "/etc/vpn-provisioning"
+sed -i "s/{adminApiToken}/${ADMIN_API_TOKEN}/" "/etc/vpn-provisioning"
+sed -i "s/{tenantId}/${TENANT_ID}/" "/etc/vpn-provisioning"
+sed -i "s/{applicationId}/${APPLICATION_ID}/" "/etc/vpn-provisioning"
+sed -i "s/{secretToken}/${${SECRET_TOKEN//\\/\\\\}//\//\\/}/" "/etc/vpn-provisioning"
```

Comment:
Here is a nested parameter as well, please fix according to https://www.shellcheck.net/wiki/SC2299